DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionDebugMenu.swift

```diff
@@ -310,14 +319,29 @@ final class NetworkProtectionDebugMenu: NSMenu {
 
     @objc func toggleEnforceRoutesAction(_ sender: Any?) {
         settings.enforceRoutes.toggle()
+
+        Task {
+            try await Task.sleep(interval: 0.1)
+            try await debugUtilities.restartAdapter()
+        }
```

Comment:
Wait a bit then restart the adapter so changes are applied.  If we try to reset immediately the changes don't make it to the system extension in time.